### ISSUE TITLE
 When we navigate from one control to another control NVDA announcing the previous control label along with focus control label.  

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14177]
### Steps to reproduce:
1. Repro Steps | Hit the URL: Hit the URL: YouTube Player Demo  \|  YouTube IFrame Player API  \|  Google Developers.
2. Turn on Screen Reader and Navigate to the controls present in that YouTube video
3. Check whether all the controls in the video are announcing correct or not.

### Actual behavior:
NVDA is announcing as:
'clickable Subscribe as Deenanath Thakur button  
Subscribe as Deenanath Thakur
Watch later as Deenanath Thakur button  
Watch later as Deenanath Thakur'

When we navigate from one control to another control screen reader announcing the previous focusable element along with the present focusable element. 
and the "Mute" and "Full screen labels are not announcing while we are navigating.  

### Expected behavior:
NVDA should just announce:
'clickable Subscribe as Deenanath Thakur button  
Watch later as Deenanath Thakur button'
When we navigate from one control to another control screen reader should announce present focusable element only, it should not announce previous focusable element along with the present focusable element.  

### System configuration
Browser: Edge Version 105.0.1343.42 (Official build) (64-bit)
Operating system: Windows 11 version 21H2 (OS Build 22000.856) 

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2022.2.3

#### Windows version:
Win 11

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

### Other questions
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

### Attachment
nvda attachment:
[nvdA.zip](https://github.com/nvaccess/nvda/files/9954253/nvdA.zip)

narrator attachment:
https://user-images.githubusercontent.com/93735775/200381416-224279dc-fe85-40bb-8c03-c44fd66f51c9.mp4

**Comment by CyrilleB79** [comment]
Hi @msftedad 

As a preliminary, I want to mention that I am not an NVAccess employee; I am just someone interested in NVDA development. And thus, I look regularly to NVDA's GitHub issue list.

I can see that you have opened many issues, which is a way to contribute to NVDA development. Many issues are related to issues of NVDA with webpages. And unfortunately, many of the issues that you have opened are [closed](https://github.com/nvaccess/nvda/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc++author%3Amsftedad+) without being resolved. Many times they are closed because NVAccess asks a minimal sample to illustrate the issue and help to understand it. And many times, the sample is not provided, maybe because when trying to make a minimal sample, you realize that the error comes from the website for which you reported the issue. And I think it will be the case again for this issue.

Since you are used to track and report issues on webpages with NVDA, could you please bother to create this minimal sample when submitting an issue related to a webpage? The minimal codepen example is required to understand the issue quickly. And this should prevent NVAccess people and yourself from losing time by reporting issues which will never be solved or which should be reported to webpage authors instead.

Of course citing the source webpage is also interesting to understand the real-world example, but the minimal sample is more helpful.

If this cause you any issue, just let us know.

Cheers,

**Comment by msftedad** [comment]
Please find the sample:
https://developers.google.com/youtube/youtube_player_demo

**Comment by seanbudd** [comment]
Can you please rewrite the steps to match the sample? The current issue has steps that are irrelevant to the sample provided.

**Comment by msftedad** [comment]
Updated the repro steps!

**Comment by msftedad** [comment]
Updated the attachment!

**Comment by Adriani90** [comment]
This occurs also in Chrome.

Note the when tabbing to the "share toggle button" and then one more tab to "Play button", NVDA reports "Share Play Button" instead of expected just "Play Button". This occurs both in focus and browse mode and only when tabbing.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by SaschaCowley** [comment]
@msftedad I can't reproduce this with NVDA alpha-53327,2274e980 (2026.1.0.53327) and Edge 143.0.3650.7 (Official build) beta (64-bit). Are you still seeing this issue?

**Comment by Adriani90** [comment]
This is not reproducible anymore in Edge, Chrome canary 144 or Firefox 145 with NVDA last alpha. I could reproduce this issue before as well.
Closing as works for me.
If you still have this problem, please comment with updated test results and we can reopen.
